docs/en/tutorials/getting-started.rst

```diff
@@ -228,33 +227,196 @@ entity definition:
+    the Reflection API and does not run either the constructor or other
+    methods.
+
+This method is mostly used when you want to focus on behavior-less
+entities, and when you need to move all your business logic in your
+services rather than in the objects themselves.
+
+It is a common convention which makes it possible to expose each field
+of your entity to external services, while allowing you to keep type
+safety in place.
+
+Such approach is a good choice for RAD (rapid application development),
+but may bring problems later down the road, because providing such an
+easy way to modify any field in your entity means the entity itself cannot
+ensure it's in valid state. Having entity in invalid state is dangerous,
+because it's one step away from being implicitly saved in database, thereby
+leading to corrupted or inconsistent data in your storage.
+
+.. note::
+
+    This method, although very common, is inappropriate for Domain Driven
+    Design (DDD), because in such design, methods should be named according
+    to actual business operations, and entities should be valid anytime.
+
+Here is an example of a simple **anemic model**:
+
+.. configuration-block::
+
+    .. code-block:: php
+
+        <?php
+        class User
         {
-            return $this->id;
+            private $username;
+            private $passwordHash;
+            private $bans;
+
+            public function getUsername(): string
+            {
+                return $this->username;
+            }
+
+            public function setUsername(string $username): void
+            {
+                $this->username = $username;
+            }
+
+            public function getPasswordHash(): string
+            {
+                return $this->passwordHash;
+            }
+
+            public function setPasswordHash(string $passwordHash): void
+            {
+                $this->passwordHash = $passwordHash;
+            }
+
+            public function getBans(): array
+            {
+                return $this->bans;
+            }
+
+            public function addBan(Ban $ban): void
+            {
+                $this->bans[] = $ban;
+            }
         }
 
-        public function getName()
+Here, we avoid all possible logic from the model and only care about injecting
+data into it without validation nor consideration about the object's state.
+
+And as Doctrine ORM is a persistence tool for your domain, the state of an
+object is really important. This is why we recommend using rich models.
+
+**Rich models: Mutators and DTOs**
+
+We recommend using a rich model design and rely on more complex
+mutators, ideally based on DTOs.
+This way, you should **not** use getters nor setters, and instead,
+implement methods that represent the **behavior** of your domain.
+
+For example, when having a ``User`` entity, we could foresee
+the following kind of optimization.
+
+Example of a rich model with proper accessors and mutators:
```

Comment:
s/model/entity

Reply:
We should clarify that this is a stub, since you keep editing it later in the documentation